[BUG] DnD5e 4.1 compatibility

**Description**
So, 4.1 just came out and I obviously don't expect RSR to be compatible right away. I just wanted to create this issue to let users share their experience and track progress on the update.

From my quick testing, RSR does not completely break with the update, but fast forwarding rolls does not work anymore. Chat cards still get merged, so not all functionality breaks.

I am currently running FoundryVTT V12 Build 331 on Forge with D&D 4.1.0.  I have two active modules running: 1) Ready Set Roll 3.3.5 and 2) The Forge 1.14.9.  

When rolling any attack from player character sheets, creatures, or NPCs the Attack Roll is rolled.  Regardless of the "Roll Damage Manually" setting, I must manually roll damage. I receive this console output when clicking the Attack Roll:

foundry.js:655 Error: Error thrown in hooked function '' for hook 'dnd5e.preUseActivity'. CONFIG.Dice.D20Roll.determineAdvantageMode is not a function
    at Hooks.onError (foundry.js:654:24)
    at #call (foundry.js:636:36)
    at Hooks.call (foundry.js:614:38)
    at AttackActivity.use (mixin.mjs:231:16)
    at Item5e.use (item.mjs:810:23)
    at InventoryElement._onAction (inventory.mjs:444:21)

--- a/lang/en.json
+++ b/lang/en.json
@@ -5,8 +5,6 @@
             "enableSkillQuickRoll.hint": "When clicking on a skill check, automatically roll to chat instead of using the normal roll dialog. This functionality can be bypassed by holding the 'Skip Dialog' key modifier when clicking.",
             "enableAbilityQuickRoll.name": "Enable Quick Roll for Abilities",
             "enableAbilityQuickRoll.hint": "When clicking on an ability check or saving throw, quick roll to chat instead of using the normal roll dialog. This functionality can be bypassed by holding the 'Skip Dialog' key modifier when clicking.",
-            "enableDeathQuickRoll.name": "Enable Quick Roll for Death Saves",
-            "enableDeathQuickRoll.hint": "When clicking on a death saving throw, quick roll to chat instead of using the normal roll dialog. This functionality can be bypassed by holding the 'Skip Dialog' key modifier when clicking.",
             "enableToolQuickRoll.name": "Enable Quick Roll for Tool Checks",
             "enableToolQuickRoll.hint": "When clicking on a tool check, quick roll to chat instead of using the normal roll dialog. This functionality can be bypassed by holding the 'Skip Dialog' key modifier when clicking.",
             "enableActivityQuickRoll.name": "Enable Quick Roll for Activities",

--- a/src/utils/activity.js
+++ b/src/utils/activity.js
@@ -15,9 +15,7 @@ export class ActivityUtility {
         
         if (!message.data.flags[MODULE_SHORT].quickRoll) {
             return;
-        }
-
-        activity._triggerSubsequentActions = function() {};
+        }        
 
         const hasAttack = activity.hasOwnProperty(ROLL_TYPE.ATTACK);
         const hasDamage = activity.hasOwnProperty(ROLL_TYPE.DAMAGE);
@@ -91,19 +89,29 @@ export class ActivityUtility {
 
     static getAttackFromMessage(message) {
         const activity = message.getAssociatedActivity();
-    
-        return activity.rollAttack(
-        { 
+
+        const usageConfig = { 
             advantage: message.flags[MODULE_SHORT].advantage ?? false,
             disadvantage: message.flags[MODULE_SHORT].disadvantage ?? false,            
             ammunition: message.flags[MODULE_SHORT].ammunition
-        }, 
-        { 
-            configure: false 
-        }, 
-        { 
-            create: false 
-        });
+        }
+
+        const dialogConfig = {
+            configure: false
+        }
+
+        const messageConfig = {
+            create: false,
+            data: {
+                flags: {}
+            }
+        }
+
+        messageConfig.data.flags[MODULE_SHORT] = {
+            quickRoll: true
+        }
+
+        return activity.rollAttack(usageConfig, dialogConfig, messageConfig);
     }
 
     static getDamageFromMessage(message) {
@@ -113,18 +121,28 @@ export class ActivityUtility {
         if (message.flags.dnd5e.scaling !== undefined) {
             activity.item.updateSource({ "flags.dnd5e.scaling": message.flags.dnd5e.scaling ?? 0 });
         }
-    
-        return activity.rollDamage(
-        {
+
+        const usageConfig = {
             isCritical: message.flags[MODULE_SHORT].isCritical ?? false,
             ammunition: actor.items.get(message.flags[MODULE_SHORT].ammunition)
-        }, 
-        { 
+        }
+
+        const dialogConfig = { 
             configure: false 
-        }, 
-        { 
-            create: false 
-        });
+        }
+
+        const messageConfig = {
+            create: false,
+            data: {
+                flags: {}
+            }
+        }
+
+        messageConfig.data.flags[MODULE_SHORT] = {
+            quickRoll: true
+        }
+    
+        return activity.rollDamage(usageConfig, dialogConfig, messageConfig);
     }
 
     static getFormulaFromMessage(message) {

--- a/src/utils/activity.js
+++ b/src/utils/activity.js
@@ -17,6 +17,8 @@ export class ActivityUtility {
             return;
         }
 
+        activity._triggerSubsequentActions = function() {};
+
         const hasAttack = activity.hasOwnProperty(ROLL_TYPE.ATTACK);
         const hasDamage = activity.hasOwnProperty(ROLL_TYPE.DAMAGE);
         const hasHealing = activity.hasOwnProperty(ROLL_TYPE.HEALING);

--- a/src/utils/core.js
+++ b/src/utils/core.js
@@ -28,26 +28,26 @@ export class CoreUtility {
     }
 
     /**
-     * Determines if a roll should be quick rolled or bypassed depending on core DND5e key modifiers.
-     * @param {Object} config The usage config passed in with the roll hook, which contains keyboard events.
-     * @returns {Boolean, Boolean, Boolean} Flags indicating if the roll should fast forward, has advantage, and has disadvantage respectively.
+     * Based on the provided event, determine if the keys are pressed to fulfill the specified keybinding.
+     * @param {Event} event    Triggering event.
+     * @param {string} action  Keybinding action within the `dnd5e` namespace.
+     * @returns {boolean}      Is the keybinding triggered?
      */
-    static processKeyModifiers(config = {}) {
-        const {isFF, advantageMode} = CONFIG.Dice.D20Roll.determineAdvantageMode({ 
-            event: config.event,
-            advantage: config.advantage,
-            disadvantage: config.disadvantage 
+    static areKeysPressed(event, action) {
+        if (!event) return false;
+        const activeModifiers = {};
+        const addModifiers = (key, pressed) => {
+            activeModifiers[key] = pressed;
+            KeyboardManager.MODIFIER_CODES[key].forEach(n => activeModifiers[n] = pressed);
+        };
+        addModifiers(KeyboardManager.MODIFIER_KEYS.CONTROL, event.ctrlKey || event.metaKey);
+        addModifiers(KeyboardManager.MODIFIER_KEYS.SHIFT, event.shiftKey);
+        addModifiers(KeyboardManager.MODIFIER_KEYS.ALT, event.altKey);
+        return game.keybindings.get("dnd5e", action).some(b => {
+            if (game.keyboard.downKeys.has(b.key) && b.modifiers.every(m => activeModifiers[m])) return true;
+            if (b.modifiers.length) return false;
+            return activeModifiers[b.key];
         });
-
-        if (isFF && advantageMode === CONFIG.Dice.D20Roll.ADV_MODE.NORMAL) {
-            return { fastForward: false, advantage: false, disadvantage: false}
-        }
-
-        return { 
-            fastForward: true,
-            advantage: advantageMode === CONFIG.Dice.D20Roll.ADV_MODE.ADVANTAGE,
-            disadvantage: advantageMode === CONFIG.Dice.D20Roll.ADV_MODE.DISADVANTAGE 
-        }
     }
 
     /**

--- a/src/utils/hooks.js
+++ b/src/utils/hooks.js
@@ -1,6 +1,7 @@
 import { MODULE_SHORT, MODULE_TITLE } from "../module/const.js";
 import { ActivityUtility } from "./activity.js";
 import { ChatUtility } from "./chat.js";
+import { CoreUtility } from "./core.js";
 import { LogUtility } from "./log.js";
 import { ROLL_TYPE, RollUtility } from "./roll.js";
 import { SETTING_NAMES, SettingsUtility } from "./settings.js";
@@ -71,65 +72,66 @@ export class HooksUtility {
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_ABILITY_ENABLED)) { 
             Hooks.on(HOOKS_DND5E.PRE_ROLL_ABILITY_CHECK, (config, dialog, message) => {
-                RollUtility.processActorRoll(config, dialog, message);
+                RollUtility.processRoll(config, dialog, message);
                 return true;
             });
 
             Hooks.on(HOOKS_DND5E.PRE_ROLL_SAVING_THROW, (config, dialog, message) => {
-                RollUtility.processActorRoll(config, dialog, message);
+                RollUtility.processRoll(config, dialog, message);
                 return true;
             });
         }
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_SKILL_ENABLED)) { 
             Hooks.on(HOOKS_DND5E.PRE_ROLL_SKILL, (config, dialog, message) => {
-                RollUtility.processActorRoll(config, dialog, message);
+                RollUtility.processRoll(config, dialog, message);
                 return true;
             });
         }
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_TOOL_ENABLED)) { 
             Hooks.on(HOOKS_DND5E.PRE_ROLL_TOOL_CHECK, (config, dialog, message) => {
-                RollUtility.processActorRoll(config, dialog, message);
+                RollUtility.processRoll(config, dialog, message);
                 return true;
             });
-        }        
-
-        // This needs to be outside if checks because it is also needed for vanilla rolls
-        // Should be removed in 4.1.0+ as this will be fixed internally by the system.
-        Hooks.on(HOOKS_DND5E.PRE_ROLL_DAMAGE, (rollConfig, dialogConfig, messageConfig) => {
-            if (!messageConfig.data.flags.dnd5e.originatingMessage) {
-                const messageId = rollConfig.event?.target.closest("[data-message-id]")?.dataset.messageId;
-                messageConfig.data.flags.dnd5e.originatingMessage = messageId;
-            }
-
-            return true;
-        });
+        }
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_ACTIVITY_ENABLED)) {
             Hooks.on(HOOKS_DND5E.PRE_USE_ACTIVITY, (activity, usageConfig, dialogConfig, messageConfig) => {              
-                RollUtility.processActivityRoll(usageConfig, dialogConfig, messageConfig);
+                RollUtility.processActivity(usageConfig, dialogConfig, messageConfig);
+                ActivityUtility.setRenderFlags(activity, messageConfig);
                 return true;
             });
 
-            Hooks.on(HOOKS_DND5E.PRE_CREATE_USAGE_MESSAGE, (activity, message) => {
-                ActivityUtility.setRenderFlags(activity, message);
-            });
-
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_ATTACK, (rollConfig, dialogConfig, messageConfig) => {
-                rollConfig.rolls[0].options.advantage = rollConfig.advantage;
-                rollConfig.rolls[0].options.disadvantage = rollConfig.disadvantage;
-                return true;
-            })
-
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_DAMAGE, (rollConfig, dialogConfig, messageConfig) => {
-                for ( const roll of rollConfig.rolls ) {
-                    roll.options ??= {};
-                    roll.options.isCritical ??= rollConfig.isCritical;
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_ATTACK, (config, dialog, message) => {
+                for (const roll of config.rolls) {
+                    roll.options.advantage ??= config.advantage;
+                    roll.options.disadvantage ??= config.disadvantage; 
                 }
 
+                const keys = {
+                    normal: CoreUtility.areKeysPressed(config.event, "skipDialogNormal")
+                };
+        
+                dialog.configure = keys.normal || (config.vanilla ?? false);
+
                 return true;
             });
+
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_DAMAGE, (config, dialog, message) => {
+                for ( const roll of config.rolls ) {
+                    roll.options ??= {};
+                    roll.options.isCritical ??= config.isCritical;
+                }
+
+                const keys = {
+                    normal: CoreUtility.areKeysPressed(config.event, "skipDialogNormal")
+                };
+        
+                dialog.configure = keys.normal || (config.vanilla ?? false);
+
+                return true;
+            });           
 
             Hooks.on(HOOKS_DND5E.ACTIVITY_CONSUMPTION, (activity, usageConfig, messageConfig, updates) => {
                 if (activity.hasOwnProperty(ROLL_TYPE.ATTACK) && updates.item.length > 0 && messageConfig.data) {

--- a/src/utils/hooks.js
+++ b/src/utils/hooks.js
@@ -97,38 +97,35 @@ export class HooksUtility {
         }
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_ACTIVITY_ENABLED)) {
-            Hooks.on(HOOKS_DND5E.PRE_USE_ACTIVITY, (activity, usageConfig, dialogConfig, messageConfig) => {              
+            Hooks.on(HOOKS_DND5E.PRE_USE_ACTIVITY, (activity, usageConfig, dialogConfig, messageConfig) => {
+                activity._triggerSubsequentActions = function() {};
                 RollUtility.processActivity(usageConfig, dialogConfig, messageConfig);
                 ActivityUtility.setRenderFlags(activity, messageConfig);
                 return true;
             });
 
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_ATTACK, (config, dialog, message) => {
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_ATTACK, (config, dialog, message) => {                
+                if (!message.data?.flags || !message.data.flags[MODULE_SHORT]?.quickRoll) return true;
+
                 for (const roll of config.rolls) {
                     roll.options.advantage ??= config.advantage;
                     roll.options.disadvantage ??= config.disadvantage; 
                 }
 
-                const keys = {
-                    normal: CoreUtility.areKeysPressed(config.event, "skipDialogNormal")
-                };
-        
-                dialog.configure = keys.normal || (config.vanilla ?? false);
+                dialog.configure = false;
 
                 return true;
             });
 
             Hooks.on(HOOKS_DND5E.PRE_ROLL_DAMAGE, (config, dialog, message) => {
+                if (!message.data?.flags || !message.data.flags[MODULE_SHORT]?.quickRoll) return true;
+
                 for ( const roll of config.rolls ) {
                     roll.options ??= {};
                     roll.options.isCritical ??= config.isCritical;
                 }
-
-                const keys = {
-                    normal: CoreUtility.areKeysPressed(config.event, "skipDialogNormal")
-                };
         
-                dialog.configure = keys.normal || (config.vanilla ?? false);
+                dialog.configure = false;
 
                 return true;
             });           

--- a/src/utils/hooks.js
+++ b/src/utils/hooks.js
@@ -11,11 +11,10 @@ export const HOOKS_CORE = {
 }
 
 export const HOOKS_DND5E = {
-    PRE_ROLL_ABILITY_TEST: "dnd5e.preRollAbilityTest",
-    PRE_ROLL_ABILITY_SAVE: "dnd5e.preRollAbilitySave",
-    PRE_ROLL_DEATH_SAVE: "dnd5e.preRollDeathSave",
-    PRE_ROLL_SKILL: "dnd5e.preRollSkill",
-    PRE_ROLL_TOOL_CHECK: "dnd5e.preRollToolCheck",
+    PRE_ROLL_ABILITY_CHECK: "dnd5e.preRollAbilityCheckV2",
+    PRE_ROLL_SAVING_THROW: "dnd5e.preRollSavingThrowV2",
+    PRE_ROLL_SKILL: "dnd5e.preRollSkillV2",
+    PRE_ROLL_TOOL_CHECK: "dnd5e.preRollToolV2",
     PRE_ROLL_ATTACK: "dnd5e.preRollAttackV2",
     PRE_ROLL_DAMAGE: "dnd5e.preRollDamageV2",
     PRE_USE_ACTIVITY: "dnd5e.preUseActivity",
@@ -71,34 +70,27 @@ export class HooksUtility {
         LogUtility.log("Registering roll hooks");
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_ABILITY_ENABLED)) { 
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_ABILITY_TEST, (actor, config, abilityId) => {
-                RollUtility.processActorRoll(config);
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_ABILITY_CHECK, (config, dialog, message) => {
+                RollUtility.processActorRoll(config, dialog, message);
                 return true;
             });
 
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_ABILITY_SAVE, (actor, config, abilityId) => {
-                RollUtility.processActorRoll(config);
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_SAVING_THROW, (config, dialog, message) => {
+                RollUtility.processActorRoll(config, dialog, message);
                 return true;
             });
         }
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_SKILL_ENABLED)) { 
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_SKILL, (actor, config, skillId) => {
-                RollUtility.processActorRoll(config);
-                return true;
-            });
-        }
-
-        if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_DEATH_ENABLED)) { 
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_DEATH_SAVE, (actor, config) => {
-                RollUtility.processActorRoll(config);
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_SKILL, (config, dialog, message) => {
+                RollUtility.processActorRoll(config, dialog, message);
                 return true;
             });
         }
 
         if (SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_TOOL_ENABLED)) { 
-            Hooks.on(HOOKS_DND5E.PRE_ROLL_TOOL_CHECK, (actor, config, toolId) => {
-                RollUtility.processActorRoll(config);
+            Hooks.on(HOOKS_DND5E.PRE_ROLL_TOOL_CHECK, (config, dialog, message) => {
+                RollUtility.processActorRoll(config, dialog, message);
                 return true;
             });
         }        

--- a/src/utils/roll.js
+++ b/src/utils/roll.js
@@ -80,6 +80,7 @@ export class RollUtility {
         };
 
         dialogConfig.configure = keys.normal || (usageConfig.vanilla ?? false);
+        usageConfig.vanilla = keys.normal;
 
         messageConfig.data.flags[MODULE_SHORT] = { 
             quickRoll: !(dialogConfig.configure ?? true),

--- a/src/utils/roll.js
+++ b/src/utils/roll.js
@@ -48,7 +48,9 @@ export const CRIT_TYPE = {
  * Utility class for functions related to making specific rolls.
  */
 export class RollUtility {
-    static processActorRoll(config, dialog, message) {
+    static processRoll(config, dialog, message) {
+        if (message.data.flags[MODULE_SHORT]?.processed) return;
+
         const keys = {
             normal: CoreUtility.areKeysPressed(config.event, "skipDialogNormal"),
             advantage: CoreUtility.areKeysPressed(config.event, "skipDialogAdvantage"),
@@ -70,18 +72,20 @@ export class RollUtility {
         };
     }
 
-    static processActivityRoll(usageConfig, dialogConfig, messageConfig) {
-        const { fastForward, advantage, disadvantage } = CoreUtility.processKeyModifiers(usageConfig);
+    static processActivity(usageConfig, dialogConfig, messageConfig) {
+        const keys = {
+            normal: CoreUtility.areKeysPressed(usageConfig.event, "skipDialogNormal"),
+            advantage: CoreUtility.areKeysPressed(usageConfig.event, "skipDialogAdvantage"),
+            disadvantage: CoreUtility.areKeysPressed(usageConfig.event, "skipDialogDisadvantage")
+        };
 
-        usageConfig.fastForward = fastForward && !usageConfig.vanilla;
-        usageConfig.advantage ||= advantage;
-        usageConfig.disadvantage ||= disadvantage;
+        dialogConfig.configure = keys.normal || (usageConfig.vanilla ?? false);
 
         messageConfig.data.flags[MODULE_SHORT] = { 
-            quickRoll: usageConfig.fastForward,
-            advantage: usageConfig.advantage,
-            disadvantage: usageConfig.disadvantage,
-            processed: !usageConfig.fastForward
+            quickRoll: !(dialogConfig.configure ?? true),
+            advantage: keys.advantage,
+            disadvantage: keys.disadvantage,
+            processed: dialogConfig.configure ?? true
         };
     }
 

--- a/src/utils/roll.js
+++ b/src/utils/roll.js
@@ -48,22 +48,24 @@ export const CRIT_TYPE = {
  * Utility class for functions related to making specific rolls.
  */
 export class RollUtility {
-    static processActorRoll(config) {
-        const { fastForward, advantage, disadvantage } = CoreUtility.processKeyModifiers(config);
+    static processActorRoll(config, dialog, message) {
+        const keys = {
+            normal: CoreUtility.areKeysPressed(config.event, "skipDialogNormal"),
+            advantage: CoreUtility.areKeysPressed(config.event, "skipDialogAdvantage"),
+            disadvantage: CoreUtility.areKeysPressed(config.event, "skipDialogDisadvantage")
+        };
 
-        config.fastForward = fastForward && !config.vanilla;
-        config.advantage ||= advantage;
-        config.disadvantage ||= disadvantage;
+        dialog.configure ??= keys.normal || (config.vanilla ?? false);
 
         if (config.isConcentration) {
             config.flavor = `${CoreUtility.localize("DND5E.ToolPromptTitle", { tool: CoreUtility.localize("DND5E.Concentration") })}`;
         }
 
-        config.messageData[`flags.${MODULE_SHORT}`] = { 
-            quickRoll: SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_VANILLA_ENABLED) || config.fastForward,
-            advantage: config.advantage,
-            disadvantage: config.disadvantage,
-            isConcentration: config.isConcentration,            
+        message.data.flags[MODULE_SHORT] = { 
+            quickRoll: SettingsUtility.getSettingValue(SETTING_NAMES.QUICK_VANILLA_ENABLED) || !(dialog.configure ?? true),
+            advantage: keys.advantage,
+            disadvantage: keys.disadvantage,
+            isConcentration: config.isConcentration,
             processed: true
         };
     }

--- a/src/utils/roll.js
+++ b/src/utils/roll.js
@@ -79,14 +79,14 @@ export class RollUtility {
             disadvantage: CoreUtility.areKeysPressed(usageConfig.event, "skipDialogDisadvantage")
         };
 
-        dialogConfig.configure = keys.normal || (usageConfig.vanilla ?? false);
-        usageConfig.vanilla = keys.normal;
+        const fastForward = !(keys.normal || (usageConfig.vanilla ?? false))
+        dialogConfig.configure = usageConfig.hasOwnProperty('spell') || !fastForward;
 
         messageConfig.data.flags[MODULE_SHORT] = { 
-            quickRoll: !(dialogConfig.configure ?? true),
+            quickRoll: fastForward,
             advantage: keys.advantage,
             disadvantage: keys.disadvantage,
-            processed: dialogConfig.configure ?? true
+            processed: !fastForward
         };
     }
 

--- a/src/utils/settings.js
+++ b/src/utils/settings.js
@@ -9,7 +9,6 @@ import { LogUtility } from "./log.js";
 export const SETTING_NAMES = {
     QUICK_SKILL_ENABLED: "enableSkillQuickRoll",
     QUICK_ABILITY_ENABLED: "enableAbilityQuickRoll",
-    QUICK_DEATH_ENABLED: "enableDeathQuickRoll",
     QUICK_TOOL_ENABLED: "enableToolQuickRoll",
     QUICK_ACTIVITY_ENABLED: "enableActivityQuickRoll",
     QUICK_VANILLA_ENABLED: "enableVanillaQuickRoll",
@@ -41,7 +40,6 @@ export class SettingsUtility {
 		const quickRollOptions = [
             { name: SETTING_NAMES.QUICK_ABILITY_ENABLED, default: true },
             { name: SETTING_NAMES.QUICK_SKILL_ENABLED, default: true },
-            { name: SETTING_NAMES.QUICK_DEATH_ENABLED, default: true },
             { name: SETTING_NAMES.QUICK_TOOL_ENABLED, default: true },
             { name: SETTING_NAMES.QUICK_ACTIVITY_ENABLED, default: true },
             { name: SETTING_NAMES.QUICK_VANILLA_ENABLED, default: false }

